src/parser.ts

```diff
@@ -149,91 +149,23 @@ export const getParser = (parser: Parser["parse"]) =>
             return getTagOrderWeight(a.tag) - getTagOrderWeight(b.tag);
           });
 
-          // add an empty line between groups
-          if (tags.length - 1 !== index) {
-            tagGroup.push(SPACE_TAG_DATA);
-          }
-
-          return tagGroup;
-        })
-        .map(
-          ({
-            type,
-            name,
-            optional,
-            default: _default,
-            description,
-            tag,
-            ...rest
-          }) => {
-            const isVerticallyAlignAbleTags = TAGS_VERTICALLY_ALIGN_ABLE.includes(
-              tag,
-            );
-
-            if (type) {
-              // Additional operations on name
-              if (name) {
-                // Optional tag name
-                if (optional) {
-                  // Figure out if tag type have default value
-                  if (_default) {
-                    description = description.replace(
-                      /[ \t]*Default is `.*`\.?$/,
-                      "",
-                    );
-                    if (description && !/[.\n]$/.test(description)) {
-                      description += ".";
-                    }
-                    description += ` Default is \`${_default}\``;
-                    name = `[${name}=${_default}]`;
-                  } else {
-                    name = `[${name}]`;
-                  }
-                }
-              }
-            }
-
-            if (isVerticallyAlignAbleTags) {
-              maxTagTitleLength = Math.max(maxTagTitleLength, tag.length);
-              maxTagTypeLength = Math.max(maxTagTypeLength, type.length);
-              maxTagNameLength = Math.max(maxTagNameLength, name.length);
-            }
-
-            description = description.trim();
-
-            return {
-              type,
-              name,
-              optional,
-              default: _default,
-              description,
-              tag,
-              ...rest,
-            };
-          },
-        )
-        .filter(({ description, tag }) => {
-          if (!description && TAGS_DESCRIPTION_NEEDED.includes(tag)) {
-            return false;
-          }
-          return true;
-        })
-        // Create final jsDoc string
-        .forEach((tagData, tagIndex, finalTagsArray) => {
-          comment.value += stringify(
-            tagData,
-            tagIndex,
-            finalTagsArray,
-            { ...options, printWidth: commentContentPrintWidth },
-            maxTagTitleLength,
-            maxTagTypeLength,
-            maxTagNameLength,
-          );
+          return tagGroup.map((tag) => {
+            combineName(tag);
+            return tag;
+          });
         });
 
-      comment.value = comment.value.trimEnd();
-      if (comment.value) {
-        comment.value = addStarsToTheBeginningOfTheLines(comment.value);
+      const content = stringifyCommentContent(groups, {
```

Comment:
1- I think that is not a good idea to move mapping to stringify don't do that.
2- stringify renamed bad, we are working on comment no need to mention it when naming variables or functions

Reply:
> move mapping to stringify

What do you mean by that? What mapping?

> stringify renamed

I did that because it's not clear what conceptual unit is being stringified. What does `stringify` do? Does it stringify a single tag, a group of tags, the whole comment, or only the content of the comment? `stringify` alone doesn't say much, so I specified what was being stringified.